pyproject.toml

```diff
@@ -1,33 +1,19 @@
 [project]
 name = "bblocks"
 version = "1.4.3"
-description = "A package with tools to download and analyse international development data. These tools are meant to be the building blocks of further analysis."
+description = "Building blocks for international development data."
```

Comment:
lets call it 'development data'. 
The international word is a bit problematic vs just development.